src/main/java/nl/dtls/fairdatapoint/api/config/RootConfig.java

```diff
@@ -38,28 +40,47 @@
 
 /**
  * Root config
- * 
+ *
  * @author Rajaram Kaliyaperumal <rr.kaliyaperumal@gmail.com>
  * @author Kees Burger <kees.burger@dtls.nl>
  * @since 2017-03-27
  * @version 0.1
  */
 @Configuration
 public class RootConfig {
+
     @Bean
     public static PropertySourcesPlaceholderConfigurer props(Environment env) {
-        PropertySourcesPlaceholderConfigurer configurer = 
-                new PropertySourcesPlaceholderConfigurer();        
-        YamlPropertiesFactoryBean yaml = new YamlPropertiesFactoryBean();        
+        PropertySourcesPlaceholderConfigurer configurer
+                = new PropertySourcesPlaceholderConfigurer();
+        YamlPropertiesFactoryBean yaml = new YamlPropertiesFactoryBean();
         Resource resource;
         if (env.containsProperty("fdpConfig")) {
             resource = new FileSystemResource(env.
                     getRequiredProperty("fdpConfig"));
         } else {
-            resource = new ClassPathResource("conf/fdpConfig.yml");
-        }        
-        yaml.setResources(resource);        
-        configurer.setProperties(yaml.getObject());        
+            resource = fdpConfig();
+        }
+        yaml.setResources(resource);
+        configurer.setProperties(yaml.getObject());
         return configurer;
     }
+
+    @Bean(name = "metadataMetrics")
+    public Map<String, String> metadataMetrics() {
+        Map<String, Object> config = yamlFactory().getObject();
+        Map<String, String> metadataMetrics = (Map) config.get("metadataMetrics");
+        return metadataMetrics;
+    }
+
+    @Bean
+    public YamlMapFactoryBean yamlFactory() {
+        YamlMapFactoryBean factory = new YamlMapFactoryBean();
+        factory.setResources(fdpConfig());
+        return factory;
```

Comment:
What happens if the user overrides the default properties by setting the `fdpConfig` environment variable? 🤔